Assets/Scripts/StandardSamples/UI/Common/SampleMenu.cs

```diff
@@ -0,0 +1,316 @@
+﻿/*
+ * Copyright (c) 2021 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#define SAMPLE_MENU_DEBUG
+
+namespace PlayEveryWare.EpicOnlineServices.Samples
+{
+    using System.Diagnostics;
+    using System.Runtime.CompilerServices;
+    using UnityEngine;
+    using UnityEngine.EventSystems;
+    using Debug = UnityEngine.Debug;
+
+    public abstract class SampleMenuWithFriends : SampleMenu
+    {
+        public enum FriendInteractionState
+        {
+            Hidden,
+            Disabled,
+            Enabled
+        }
+
+        protected SampleMenuWithFriends(bool startsHidden = true) : base(startsHidden) { }
```

Comment:
Feels like this should pass in a "true" to the optional parameter of the base class, although that'll be implicitly true anyways.

Reply:
Good catch! Since changing it to be `true` is the same as not defining the constructor at all, I just removed it altogether. Makes a lot more sense.